webapp/channels/src/sass/components/_post.scss

```diff
@@ -1475,7 +1475,7 @@
         }
 
         p + p {
-            margin: 0.5em 0 0;
+            margin: 0 0 0;
```

Comment:
Won't this change change how every posts will look like? Is this something desired?